include/sundials/sundials_memory.h

```diff
@@ -18,37 +18,37 @@
 #define _SUNDIALS_MEMORY_H
 
 #include <stdlib.h>
-
 #include <sundials/sundials_context.h>
 #include <sundials/sundials_types.h>
 
-#ifdef __cplusplus  /* wrapper to enable C++ usage */
+#include "sundials_linearsolver.h"
+
```

Comment:
I'm surprised that `sundials_memory.h` at all depends on `sundials_linearsolver.h` (and I also wonder why this `#include` line is missing `sundials/`).  I'll include a suggestion that this be removed so that it can be reverted in one click, in case this addition was a mistake.
```suggestion
```

Reply:
Probably was a mistake